code/__HELPERS/mobs.dm

```diff
@@ -189,6 +189,9 @@ GLOBAL_LIST_EMPTY(species_list)
 
 	var/target_loc = target.loc
 
+	LAZYADD(user.do_afters, target)

```

Comment:
This change can stand on its own PR to diminish a bit the size of this one.
Could you atomize it?

Reply:
wait, does this keep you from queueing up multiple do_afters on the same thing?

Reply:
only if you manually check IS_INTERACTING_WITH

I'll atomize it tho